Python doesn't allow a function called from a function to be defined in a different file.
